scan-results.js

```diff
@@ -86,6 +86,8 @@ const scanResult = async(req, selfScan=false) => {
     }
   }
 
+  const newScanPage = ((Math.random() * (10 - 1) + 1) >= 5);
```

Comment:
Do we need the `-1` and `+1`? Why not just `Math.random() * 10 >= 5`

Reply:
Does this mean I could get a different UI on each page reload?

Reply:
@pdehaan Yes - you could very feasibly get a different UI on reload or subsequent scans.

Reply:
I don't know if it matters or not (it's above my pay grade).
I do recall that the FxA team had built a little A/B testing thing in the past, but not sure if it's still used or if they moved onto something else: https://github.com/dannycoates/able

It might depend on how many A/B tests we plan on doing in the future and if keeping people "in" the experiment matters or not.